### PR TITLE
Test sorting on multivalue attributes.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

**Review by geirst — DISMISSED** [review]
👍

File: tests/search/sorting/sorting_multivalue.rb [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  end
+
+  def check_sorted_multi(sortspec, exp_ids)
+    for suffix in ['', '_fs', '_wset', '_wset_fs', '_s', '_s_wset']
+      check_sorted(sortspec + suffix, exp_ids)
+    end
+  end
+
+  def check_sorted(sortspec, exp_ids)
+    yql = 'select * from sources * where sddocname contains "multivalue"'
```
[the remark this answers is not in the document]
Comment (toregge) [reply tests/search/sorting/sorting_multivalue.rb]: Done.

File: tests/search/sorting/sorting_multivalue.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    form = [['yql', yql],
+            ['hits', '10']]
```
[the remark this answers is not in the document]
Comment (toregge) [reply tests/search/sorting/sorting_multivalue.rb]: Removed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**